src/main/groovy/org/mockito/release/internal/gradle/ReleaseConfigurationPlugin.java

```diff
@@ -43,10 +57,66 @@ public void apply(Project project) {
         }
     }
 
+    private void loadShipKitConfigFile(Project rootProject) {
+        final File configFile = rootProject.file(CONFIG_FILE_RELATIVE_PATH);
+        if (!configFile.exists()) {
+            createShipKitConfigFile(configFile);
+            throw new GradleException("Config file created at " + configFile.getAbsolutePath() + ". Please configure it and rerun the task.");
+        } else {
+            rootProject.apply(new Action<ObjectConfigurationAction>() {
```

Comment:
Thank you for updating the code so that it uses Java types instead of the Map!

Can you update the Javadoc of this plugin class to explain how we handle the configuration? This way we will have comprehensive documentation.